src/components/SubmissionForm/ProjectForm.js

```diff
@@ -45,11 +46,20 @@ const ProjectForm = props => {
     let submitted_at = moment().format("MMMM Do YYYY, h:mm:ss a")
     let display_image = props.display_image && props.display_image
     let newPost = {
-      ...state,
+      //send all of state to BE except category_name and error_message (will return a 500 if you include these)
```

Comment:
Feel free to remove the category_name property from state and props if we no longer have a use case for it 👍

Reply:
Removed!